### PR TITLE
Fixes Ian being racist when he wears blue wizard hats.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Fixes Ian being racist when he wears blue wizard hats.

## Why It's Good For The Game

Looks like we forgot this one.

## Changelog

:cl:
spellcheck: Fixes Ian being racist when he wears blue wizard hats.
/:cl:

### CODE REVIEW

**Comment by Farquaar** [comment]
Greybeards are dwarves, not wizards

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
goof you are off your rocker

**Comment by LemonInTheDark** [comment]
bad justification, come up with something better

**Comment by Toastgoats** [comment]
Me and some people on another discord server are trying to figure out how the hell "greybeard" is a slur, please enlighten us.

**Comment by Iamgoofball** [comment]
@Toastgoats you have it flipped around, you're looking at the new change

File: code/datums/dog_fashion.dm [hunk 1]
```diff
@@ -148,7 +148,7 @@
 	emote_see = list("drools.","looks for donuts.")
 
 /datum/dog_fashion/head/blue_wizard
-	name = "Grandwizard %REAL_NAME%"
+	name = "Warlock %REAL_NAME%"
```
Comment (LemonInTheDark) [inline code/datums/dog_fashion.dm:151]: Greybeard was better I would merge greybeard

**Comment by optimumtact** [comment]
image

**Comment by atlasle** [comment]
> goof you are off your rocker 

genuinely when is he not

**Pull request closed without merging**